packages/cli-kit/src/public/node/node-package-manager.ts

```diff
@@ -524,7 +523,7 @@ async function getLatestNPMPackageVersion(name: string) {
 }
 
 interface FindPackageVersionUpOptions {
-  fromModuleURL: URL | string
+  fromModuleURL: import('url').URL | string
```

Comment:
Curious why this is necessary

Reply:
Same as in another comment the version of `@types/node` that we are using now doesn't expose the URL type globally so we need to import it. Note that we are importing the type so it doesn't have any impact at runtime.